CHANGELOG.md

```diff
@@ -1,4 +1,124 @@
-# Change Log
+# Changelog
+
+## 2016-10-25 Release [0.5.0](https://github.com/voxpupuli/puppet-nginx/tree/0.5.0)
```

Comment:
Based on the other examples, looks like a 'v' is expected before the version number. rebased and force-pushed.